### ISSUE TITLE
Make all of src/webgpu/api/operation use max limits

### ISSUE THREAD

**Issue by greggman** [gpuweb/cts#3363]
IIUC, the current CTS doesn't test max limits operationally. It only tests the default limits and it tests validation for max limits. 

Would it be a good idea to make a MaxLimitTest that requires all max limits and have all relevant tests under src/webgpu/api/operation use that so more tests the maximum limits?

WebGL always tested maximum limits. I don't know what bugs, if any, to expect, but, it's at the max limits where the edge cases are. 

I don't think we have the capacity to write a whole bunch of new tests nor do we want to run so many tests twice, once at default and once at max. So, maybe something like just making a bunch of tests in the operation subtree use max limits would more thoroughly test?

**Comment by greggman** [comment]
Thoughts? @kainino0x @kdashg @Kangz @mwyrzykowski

**Comment by mwyrzykowski** [comment]
Seems reasonable to me

**Comment by kainino0x** [comment]
yes, I think we should. Having it be enabled on a per test basis is probably good, rather than doing this globally for operation tests, because some existing tests could break (or lose coverage) otherwise.
